packages/shared/components/popups/Transaction.svelte

```diff
@@ -28,9 +28,11 @@
     }
 
     $: accountOverview = $participationOverview?.find((apo) => apo?.accountIndex === $selectedAccount.index)
-    $: isAccountVoting =
-        Object.values(accountOverview?.trackedParticipations)?.find((tp) => tp?.find((p) => p?.endMilestoneIndex === 0))
-            .length > 0
+    $: isAccountVoting = accountOverview
+        ? Object.values(accountOverview.trackedParticipations)?.find((tp) =>
```

Comment:
You could use the nullish coalescing operator here (`??). Also whats the difference between this code and the following: 
```
$: isAccountVoting = accountOverview.trackedParticipations?.find((tp) =>
              tp?.find((p) => p?.endMilestoneIndex === 0)
          )?.length > 0 ?? false
```

Reply:
Not sure if `Object.values()` is needed here

Reply:
It's needed because trackedParticipations is an object, and objects don't have the `find` method

Reply:
> You could use the nullish coalescing operator here (`??`). Also whats the difference between this code and the following:
> 
> ```
> $: isAccountVoting = accountOverview.trackedParticipations?.find((tp) =>
>               tp?.find((p) => p?.endMilestoneIndex === 0)
>           )?.length > 0 ?? false
> ```

Awesome suggestion, that's the first time that I'm going to use it into a real project

Reply:
> You could use the nullish coalescing operator here (`??`). Also whats the difference between this code and the following:
> 
> ```
> $: isAccountVoting = accountOverview.trackedParticipations?.find((tp) =>
>               tp?.find((p) => p?.endMilestoneIndex === 0)
>           )?.length > 0 ?? false
> ```

Nice suggestion!!